.github/workflows/config/auto-assign.yaml

```
@@ -2,4 +2,4 @@ runOnDraft: false
 addReviewers: true
 addAssignees: author
 reviewers:
```

actionlint (1.7.4)

unexpected key "reviewers" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)